chains/manager.go

```diff
@@ -490,13 +490,15 @@ func (m *manager) buildChain(chainParams ChainParameters, sb subnets.Subnet) (*c
 		return nil, fmt.Errorf("error while creating chain's log %w", err)
 	}
 
+	pubKey := m.StakingBLSKey.PublicKey()
+
 	ctx := &snow.ConsensusContext{
 		Context: &snow.Context{
 			NetworkID:       m.NetworkID,
 			SubnetID:        chainParams.SubnetID,
 			ChainID:         chainParams.ID,
 			NodeID:          m.NodeID,
-			PublicKey:       m.StakingBLSKey.PublicKey(),
+			PublicKey:       pubKey,
```

Comment:
I think we can undo this diff now that `PublicKey` isn't erroring (and in a bunch of other spots in this PR)